### PR TITLE
Record FIB and RIB programming status of AFT entries in GetResponse.  

### PR DESCRIPTION
**Pull request by xw-g**
Implement the conclusion of issue #44 .

### CODE REVIEW

**Review by ashpatcisco — APPROVED** [review]

File: v1/proto/service/gribi.proto [hunk 1]
```diff
@@ -465,6 +465,18 @@ message AFTEntry {
...
+  enum Status {
+    // The programming status is not available.
+    // When the session parameter is set to RIB ack only, the device might
+    // not keep track of FIB programming status.
+    UNAVAILABLE = 0;
+    PROGRAMMED = 1;
+    NOT_PROGRAMMED = 2;
+  }
+  // RIB programming status of the entry.
+  Status rib = 11;
```
Comment (nandanarista) [inline v1/proto/service/gribi.proto:477]: How about `rib_status` instead of just `rib`?
Likewise `fib_status`

File: v1/proto/service/gribi.pb.go [hunk 2]
```diff
@@ -635,7 +687,15 @@ type ModifyRequest struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// A group of requests to add/modify/remove a single AFT entry
+	// A group of requests to add/modify/remove a single AFT entry.
```
Comment (nandanarista) [inline v1/proto/service/gribi.pb.go:690]: there are many changes to the comments but they're not in the .proto file corresponding to this.
Where are these changes coming from? I assume this file hasn't been hand edited

**Review by nandanarista — COMMENTED** [review]
LGTM, a suggestion on the variable names and a qn on some changes in the generated go files

File: v1/proto/service/gribi.pb.go (hunk 2 above)
Comment (nandanarista) [reply v1/proto/service/gribi.pb.go:690]: resolving this, they're coming from prior merges that were comment only.

File: v1/proto/service/gribi.proto (hunk 1 above)
Comment (xw-g) [reply v1/proto/service/gribi.proto]: Done

**Review by nandanarista — APPROVED** [review]
LGTM

**Pull request merged**